corehq/tabs/uitab.py

```diff
@@ -149,7 +149,11 @@ def url(self):
 
     @property
     def url_prefixes(self):
-        return [url_prefix_format.format(domain=self.domain)
+        # Use self._request.domain instead of self.domain to generate url-prefixes
+        # because the latter might have a normalized domain name which might not match the
+        # domain name mentioned in the URL. for example domain-name 'hq_test' is normalized to
+        # 'ht-test'
```

Comment:
Yeah, I thought about it, but replacing would break things that query the db by domain-name, (the normalized domain name is the name stored in db). Further this issue is a rare accident. Someone using hq_test instead of hq-test in URL is rare (as all the urls generated by HQ will have the latter).